packages/document/builder-doc/docs/en/config/experiments/sourceBuild.md

```diff
@@ -1,6 +1,7 @@
 - **Type:** `boolean`
 - **Default:** `false`
 - **Bundler:** `only support webpack`
+- **Version:** `2.26.0`
```

Comment:
We can use `MAJOR_VERSION` as the placeholder, it will be replaced to `2` for Modern.js and `1` for EdenX.

```suggestion
- **Version:** `MAJOR_VERSION.26.0`
```